FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -77,7 +77,14 @@
 #import "FirebaseAuth/Sources/SystemService/FIRAuthAppCredentialManager.h"
 #import "FirebaseAuth/Sources/SystemService/FIRAuthNotificationManager.h"
 #import "FirebaseAuth/Sources/Utilities/FIRAuthURLPresenter.h"
-#endif
+
+#if SWIFT_PACKAGE
```

Comment:
Structure public headers as documented in https://github.com/firebase/firebase-ios-sdk/blob/master/HeadersImports.md#header-file-types-and-locations---for-header-file-creators to avoid need for SPM/CocoaPods specific imports.

Reply:
Hm, not sure I understand how it's applicable here because there are no headers in the lib currently, but only a single Swift file. But probably it won't be relevant any more after https://github.com/google/GoogleUtilities/pull/50#discussion_r674200503 since we will have to re-implement the multicast delegate in Objective-C.

Reply:
Oh right. I made this comment before I saw the implementation was Swift.

Reply:
TODO: Validate in Firebase